core/src/main/java/feast/core/grpc/CoreServiceImpl.java

```diff
@@ -68,110 +69,52 @@ public CoreServiceImpl(
   public void getFeastCoreVersion(
       GetFeastCoreVersionRequest request,
       StreamObserver<GetFeastCoreVersionResponse> responseObserver) {
-    try {
-      GetFeastCoreVersionResponse response =
-          GetFeastCoreVersionResponse.newBuilder().setVersion(feastProperties.getVersion()).build();
-      responseObserver.onNext(response);
-      responseObserver.onCompleted();
-    } catch (RetrievalException | StatusRuntimeException e) {
-      log.error("Could not determine Feast Core version: ", e);
-      responseObserver.onError(
-          Status.INTERNAL.withDescription(e.getMessage()).withCause(e).asRuntimeException());
-    }
+    GetFeastCoreVersionResponse response =
```

Comment:
This code looks much better with the exception handling in the interceptor, but I don't think its in scope for this PR is it?

Reply:
Yeah, moving out of this PR.